### PR TITLE
feat: add core network

### PR DESCRIPTION
**Pull request by Reinis-FRP**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
   "dependencies": {
     "@gnosis.pm/safe-apps-sdk": "^7.8.0",
     "@gnosis.pm/safe-deployments": "^1.19.0",
-    "@gnosis.pm/zodiac": "^3.3.6",
+    "@reinis_frp/zodiac": "^4.0.2",
```
Comment (Reinis-FRP) [inline package.json:18]: This is forked zodiac adding Core network. Once https://github.com/gnosisguild/zodiac/pull/157 is merged and released, we can revert back to original upstream

File: src/app/customChains.tsx
Comment (Reinis-FRP) [inline src/app/customChains.tsx:1]: We currently import from wagmi/chains that stopped updates. After we import directly from viem that has coreDao added, we can drop this

File: src/components/OsnapCard.tsx [hunk 2]
```diff
@@ -16,8 +16,9 @@ import {
 } from "./AdvancedSettingsModal";
 export function useOsnapCard() {
   const searchParams = useSearchParams();
-  const spaceName = searchParams.get("spaceName") ?? undefined;
-  const spaceUrl = searchParams.get("spaceUrl") ?? undefined;
+  const spaceName = searchParams.get("spaceName") ?? "uma dev";
+  const spaceUrl =
+    searchParams.get("spaceUrl") ?? "https://snapshot.org/#/umadev.eth";
```
Comment (Reinis-FRP) [inline src/components/OsnapCard.tsx:21]: These are just hardcoded defaults for testing. We need to add form to pass spaceName and spaceUrl by user if not present in URL search params.

File: src/components/OsnapCard.tsx [hunk 3]
```diff
@@ -152,8 +152,8 @@ export function OsnapCard() {
     advancedSettingsModalProps.showModal();
   }
 
-  function activateOsnap() {
-    deploy?.();
+  async function activateOsnap() {
+    await deploy?.();
```
Comment (daywiss) [inline src/components/OsnapCard.tsx:156]: this should be `deploy?.().catch(err=>console.error("Error Deploying:",err)`

Comment (Reinis-FRP) [reply src/components/OsnapCard.tsx]: fixed, though linter complaind and had to use more verbose version with await

**Review by daywiss — APPROVED** [review]
ok looks good. we can revisit zodiac deps later

**Pull request merged**